doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
+
+We aim to demonstrate the standard orchestrator automation capabilities of SmartSim, so we
+create a single database in the workflow: a multi-sharded database.
+
+Step 1: Initialize Orchestrator
+'''''''''''''''''''''''''''''''
+To create a standard database, utilize the ``Experiment.create_database()`` function.
+.. code-block:: python
+
+  # Initialize a multi sharded database
+  standard_db = exp.create_database(db_nodes=3)
+  exp.generate(standard_db)
+
+Step 2: Start Databases
+'''''''''''''''''''''''
+Next, to launch the database,
+pass the database instance to ``Experiment.start()``.
+.. code-block:: python
+
+  # Launch the multi sharded database
+  exp.start(standard_db)
+
+The ``Experiment.start()`` function launches the ``Orchestrator`` for use within the workflow.
+In other words, the function deploys the database on the allocated compute resources.
+
+Create Client Connections to Orchestrator
+-----------------------------------------
+The SmartRedis ``Client`` object contains functions that manipulate, send, and receive
+data within the database. Each database can have a single, dedicated SmartRedis ``Client``.
+Begin by initializing a SmartRedis ``Client`` object for the standard database.
+
+When creating a client connect from within a driver script,
+you need to specify the address of the database you would like to connect to.
+You can easily retrieve this address using the ``Orchestrator.get_address()`` function:
+
+.. code-block:: python
+
+  # Initialize a SmartRedis client for multi sharded database
+  driver_client_standard_db = Client(cluster=True, address=standard_db.get_address()[0])
+
+Store Data Using Clients
+------------------------
+In the application script, we retrieved a NumPy tensor.
+To support the apps functionality, we create a
+NumPy array in the python driver script to send to the a database. To
+accomplish this, we use ``Client.put_tensor()``:
+.. code-block:: python
+
+  # Create NumPy array
+  array_1 = np.array([1, 2, 3, 4])
+  # Use multi shard db SmartRedis client to place tensor standard database
+  driver_client_standard_db.put_tensor("tensor_1", array_1)
+
+Initialize a Model
+------------------
+In the next stage of the experiment, we
+launch the application script by configuring and creating
+a SmartSim ``Model``.
 
-A colocated Orchestrator is a special type of Orchestrator that is deployed on
-the same compute hosts an a ``Model`` instance defined by the user. In this
-deployment, the database is *not* connected together in a cluster and each
-shard of the database is addressed individually by the processes running
-on that compute host.
+Step 1: Configure
+'''''''''''''''''
+You can specify the run settings of a model.
+In this experiment, we invoke the Python interpreter to run
+the python script defined in section: The Application Script.
+To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
```

Comment:
`To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.` seems to imply that it turns run settings into a model or that a model isn't a model without run settings. It just seems unclear.

Perhaps something like "We use ``Experiment.create_run_settings()`` to create a configuration object that will define the operation of a ``Model``"